packages/turf-dissolve/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@turf/dissolve",
+  "version": "3.7.5",
+  "description": "Dissolves polygons based on an attribute",
+  "main": "index.js",
```

Comment:
- Add `types: "index.d.ts"` once the`index.d.ts` is created.

Use this module as an example:
https://github.com/Turfjs/turf/tree/master/packages/turf-bearing